readme.md

```diff
@@ -280,6 +280,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[Fluxguard](https://fluxguard.com/)** - Fluxguard provides website DOM change monitoring orchestrated with Google Puppeteer, and audited by Lighthouse. Fluxguard is a freemium product, with monthly monitoring of up to 75 pages for free.
 
+* **[Foo](https://www.foo.software)** - Foo continuously captures performance audits with Lighthouse, notifies of change, and provides visualization. Users can choose to be notified via email, Slack and PagerDuty. A dashboard with charts show changes by day, week or month and provide detailed reports.
```

Comment:
Could you add the note that this is a paid product? It also isn't clear if there is any sort of free trial.

Reply:
Thanks for reviewing @patrickhulce. Updated in 9065b24. If you had different phrasing in mind - feel free to let me know.